src/microfrontends/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.62.0",
+  "version": "1.62.1",
```

Is this a bug fix? It looks like the ticket is for a feature.
```suggestion
  "version": "1.63.0",
```

---

@knagurski  Its feature , we are migrating srm module to micro frontend ,